ddtrace/contrib/flask/patch.py

```diff
@@ -467,25 +469,7 @@ def _traced_request(pin, wrapped, instance, args, kwargs):
         if not pin.enabled or not span:
             return wrapped(*args, **kwargs)
 
-        try:
-            request = flask._request_ctx_stack.top.request
-
-            # DEV: This name will include the blueprint name as well (e.g. `bp.index`)
-            if not span.get_tag(FLASK_ENDPOINT) and request.endpoint:
-                span.resource = u" ".join((request.method, request.endpoint))
-                span._set_str_tag(FLASK_ENDPOINT, request.endpoint)
-
-            if not span.get_tag(FLASK_URL_RULE) and request.url_rule and request.url_rule.rule:
-                span.resource = u" ".join((request.method, request.url_rule.rule))
-                span._set_str_tag(FLASK_URL_RULE, request.url_rule.rule)
-
-            if not span.get_tag(FLASK_VIEW_ARGS) and request.view_args and config.flask.get("collect_view_args"):
-                for k, v in request.view_args.items():
-                    # DEV: Do not use `_set_str_tag` here since view args can be string/int/float/path/uuid/etc
-                    #      https://flask.palletsprojects.com/en/1.1.x/api/#url-route-registrations
-                    span.set_tag(u".".join((FLASK_VIEW_ARGS, k)), v)
-        except Exception:
-            log.debug('failed to set tags for "flask.request" span', exc_info=True)
+        _set_request_tags(span)
```

Comment:
Can we add a comment here saying that this call might be unnecessary since we set the tags earlier and we just haven't been able to confirm/validate?